lisa/sut_orchestrator/azure/common.py

```diff
@@ -400,6 +400,30 @@ def get_data_disk_size(
         raise LisaException("No data disk feature present.")
 
 
+@dataclass_json()
+@dataclass
+class DataDiskSchema:
+    caching_type: str = field(
+        default=constants.DATADISK_CACHING_TYPE_NONE,
+        metadata=schema.metadata(
+            validate=validate.OneOf(
+                [
+                    constants.DATADISK_CACHING_TYPE_NONE,
+                    constants.DATADISK_CACHING_TYPE_READONLY,
+                    constants.DATADISK_CACHING_TYPE_READYWRITE,
+                ]
+            )
+        ),
+    )
+    size: int = 32
+    sku: str = field(
```

Comment:
sku might be confusing variable name. Maybe use `disk_type`?

Reply:
fixed